autopush/utils.py

```diff
@@ -179,7 +180,7 @@ def extract_jwt(token, crypto_key):
     # first split and convert the jwt.
     if not token or not crypto_key:
         return {}
-    return jwt.decode(token, decipher_public_key(crypto_key))
+    return jwt.decode(token, decipher_public_key(crypto_key.encode('utf8')))
```

Comment:
No, the problem is that the base64 decoder which gets called later rejects strings, so we recode it for an accepted value.